tsconfig.json

```diff
@@ -66,7 +66,7 @@
     "skipLibCheck": true /* Skip type checking of declaration files. */,
     "forceConsistentCasingInFileNames": true, /* Disallow inconsistently-cased references to the same file. */
 
-    /* https://github.com/jaydenseric/graphql-upload/issues/282#issuecomment-1142691771 */
+    /* https://github.com/flash-oss/graphql-upload-minimal/issues/282#issuecomment-1142691771 */
```

Comment:
This seems wrongly renamed, could you check?

Reply:
yes I remove the comment is useless in this case